cobigen/cobigen-core-systemtest/src/test/java/com/devonfw/cobigen/systemtest/TemplatesInstallationTest.java

```diff
@@ -0,0 +1,130 @@
+package com.devonfw.cobigen.systemtest;
+
+import static com.github.stefanbirkner.systemlambda.SystemLambda.withEnvironmentVariable;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+
+import org.junit.Test;
+
+import com.devonfw.cobigen.api.constants.ConfigurationConstants;
+import com.devonfw.cobigen.impl.CobiGenFactory;
+import com.devonfw.cobigen.systemtest.common.AbstractApiTest;
+
+/**
+ * Test suite for template installation with the cobigen startup
+ *
+ */
+public class TemplatesInstallationTest extends AbstractApiTest {
+
+  /**
+   * Tests if the templates specified in the .cobigen file will be loaded at startup with the template-set structure and
+   * an existing downloaded folder.
+   *
+   * TODO: Check if this test is valid
```

Comment:
Create a issue and reference it or remove the TODO

Reply:
added.